zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -83,19 +83,65 @@
               </span>
             </div>
           </form>
-        <li class="server-status"  >
+        <li class="server-status">
+        <div class="dropdown">
           <i class="fa fa-circle" ng-class="{'server-connected':navbar.connected, 'server-disconnected':!navbar.connected}"></i>
-        <span ng-show="navbar.connected" ng-if="ticket.principal == 'anonymous' ">Connected</span>
-        <span ng-show="navbar.connected" ng-if="ticket.principal != 'anonymous' " tooltip-placement="bottom" tooltip="{{fullUsername}}">{{truncatedUsername}}</span>
-        <span ng-show="!navbar.connected">Disconnected</span>
+          <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown"
+                  ng-show="navbar.connected" ng-if="ticket.principal == 'anonymous' ">Connected
+            <span class="caret"></span></button>
+          <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown"
+                  ng-show="navbar.connected" ng-if="ticket.principal != 'anonymous' "
+                  tooltip-placement="bottom" tooltip="{{fullUsername}}">{{truncatedUsername}}
+            <span class="caret"></span></button>
+          <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown"
+                  ng-show="!navbar.connected">Disconnected
+            <span class="caret"></span></button>
+          <ul class="dropdown-menu">
+            <li><a href="" data-toggle="modal" data-target="#aboutModal" >
+              <i style="font-size: 15px;" class="fa fa-info"></i> About</a></li>
+            <li ng-show="ticket.principal && ticket.principal!='anonymous'" style="left: 5px;">
+              <a  ng-click="logout()" tooltip-placement="bottom" tooltip="logout">Logout</a>
```

Comment:
@prabhjyotsingh In my personal opinion, we don't need another tooltip on **Logout** button. Users can aware that this is logout button without tooltip i think :)
<img width="194" alt="screen shot 2016-06-02 at 9 42 33 am" src="https://cloud.githubusercontent.com/assets/10060731/15753189/a6c630a6-28a6-11e6-958e-0294437a71e6.png">